client/package.json

```diff
@@ -4,8 +4,12 @@
   "description": "the client app for the sim.",
   "main": "src/index.js",
   "scripts": {
+    "start": "./node_modules/.bin/http-server . -p 1337",
```

Comment:
node should automatically make all packages avaliable.  Try this:
`"start": "http-server -p 1337"`  (current location is implied as root path if not provided)
See if that works.

Reply:
Ah! That is perfect. I did not know that was possible.